digital_land/datatype/wkt.py

```diff
@@ -182,10 +183,14 @@ def normalise_geometry(geometry, simplification=0.000005):
 
 
 def dump_wkt(geometry, precision=6, dimensions=2):
+    try:
+        current_precision = len(str(x).split(".")[1]) if "." in str(x) else 0
+    except Exception:
+        current_precision = 0
     wkt = shapely.wkt.dumps(
         geometry, rounding_precision=precision, output_dimension=dimensions
     )
-    return wkt.replace(", ", ",")
+    return wkt.replace(", ", ","), True if current_precision > 6 else False
```

Comment:
I'm worried that this is going to be very late in the process, i.e. we've made a lot of changes by now so the precision issue could be caused by our transformations etc. rather than them providing a geometry that's too precise this issue would also show that we transformed it to WGS84 and it's too precise, which while it is useful still to us it may confuse them if we showed them the issue. 

We probably need a check which happens at the start to inform us whether they are providing the geometry as too precise, and maybe one nearer the end that tells us if precision introduced during processing? Not 100% sure if we need two or not.

I'm also wondering whether we need this to happen in the dump_wkt function if it's something that can be checked somewhere else in the wkt.normalise method

Reply:
I have added this at the start of wkt.normalise and removed it from dump_wkt as it being called multiple times.
Currently it just says 'geometry provided is too precise - fixed' for when geometry is greater than 6 d.p. 
Can also rename to 'handled by pipeline' instead of 'fixed'.

How can we check at the end to make sure precision has not been introduced?